### PR TITLE
updated ettercap script to support target syntax for kali1 and kali r…

### PR DESCRIPTION
**Pull request by awk9000**
…olling versions

### CODE REVIEW

**Comment by ZeroChaos-** [comment]
Use "dpkg --list ettercap-common" instead of chainjng grep

**Comment by ZeroChaos-** [comment]
Use RETCODE instead of confusing "ver"

**Comment by bclune** [comment]
You can also `grep -q` instead of `&>/dev/null`

File: scripts/ettercap.sh [hunk 1]
```diff
@@ -68,6 +80,41 @@ f_run(){
...
+  printf "\n"
+  read -p "Enter target IP to ARP cache poison: " target1
+  printf "\n"
+
+  clear
+  printf "\n"
+  read -p "Enter target IP of gateway/router: " gw
+  printf "\n"
+
+  #ettercap fails if the interface is down
```
Comment (granolocks) [inline scripts/ettercap.sh]: do these values need any kind of validation? what happens if you input a bad value?

Comment (ZeroChaos-) [inline scripts/ettercap.sh]: it will fail if you don't plug in an IP but right now we are concentrating on critical bug fixes only so that gets a pass.

Comment (granolocks) [inline scripts/ettercap.sh]: word, will it at least dump the failure to screen?

Comment (ZeroChaos-) [inline scripts/ettercap.sh]: yes, ettercap validates the ip and makes fun of you for not typing a valid ip.  it says "Invalid ipv6 address" but close enough for now.

File: scripts/ettercap.sh [hunk 2]
```diff
@@ -39,7 +39,19 @@ f_generate_filename(){
   printf "/opt/pwnix/captures/ettercap/ettercap$(date +%F-%H%M)\n"
 }
 
-f_run(){
+f_version_check(){
+   # Check for Kali1 version for target syntax change...
+  dpkg --list |grep ettercap |grep -qi 1:0.8.2-2~kali1
```
Comment (ZeroChaos-) [inline scripts/ettercap.sh]: I said "dpkg --list ettercap-common | grep -q 1:0.8.2-2~kali1"
don't chain grep, there is no need, and don't need case insensitive grep.

**Comment by awk9000** [comment]
Just tested latest commit, working on both Nexus 9 (kali rolling) and Nexus 7 (kali 1)

**Comment by ZeroChaos-** [comment]
tested fine here too

**Pull request merged**